chroma-manager/tests/integration/utils/chroma_log_collector.py

```diff
@@ -120,67 +142,97 @@ def fetch_log(self, server, source_log_path, destination_log_filename):
         return None
 
     def fetch_log_dir(self, server, dir):
-        """
-        Collect the log directory from the target
+        """Collect the log directory from the target.
+
         :return: None on success or error message that can be used for diagnosing what went wrong.
+
         """
         logs = shell_run(['ssh', server, "ls %s | xargs -n1 basename" % dir])
```

Comment:
Is this actually `logs = shell_run(['ssh', server, "ls %s | sed -e "s/.*\//" % dir])`?